projects/migrations/0009_update_publication.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2021-05-25 21:44
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('projects', '0008_auto_20200708_1116'),
+    ]
+
+    operations = [
+        migrations.RenameField(
+            model_name='publication',
+            old_name='booktitle',
+            new_name='forum',
+        ),
+        migrations.RemoveField(
+            model_name='publication',
+            name='abstract',
+        ),
+        migrations.RemoveField(
+            model_name='publication',
+            name='journal',
+        ),
+        migrations.RemoveField(
+            model_name='publication',
+            name='publisher',
+        ),
+        migrations.AddField(
+            model_name='publication',
+            name='link',
+            field=models.CharField(max_length=500, null=True),
+        ),
+        migrations.AddField(
+            model_name='publication',
+            name='month',
+            field=models.IntegerField(null=True),
+        ),
+        migrations.AddField(
+            model_name='publication',
+            name='publication_type',
+            field=models.CharField(default='article', max_length=50),
+            preserve_default=False,
+        ),
+    ]
```

Comment:
Given that we're removing fields and storing them elsewhere, we should include a data migration. See [here](https://github.com/ChameleonCloud/portal/blob/master/sharing_portal/migrations/0010_migrate_zenodo_depositions.py) for an example.